translations/es.php

```diff
@@ -30,25 +30,25 @@
 $_MODULE['<{psgdpr}prestashop>htmltemplatepsgdprmodule_9ad5a301cfed1c7f825506bf57205ab6'] = 'DATOS PERSONALES';
 $_MODULE['<{psgdpr}prestashop>customerpersonaldata17_c654dacdea4ad6f75ad36553a8a947cd'] = 'Mis datos personales';
 $_MODULE['<{psgdpr}prestashop>customerpersonaldata17_a8308387f3ffc2d045d2b6125e74317d'] = 'Acceder a mis datos';
-$_MODULE['<{psgdpr}prestashop>customerpersonaldata17_915f4446a47b51378e37374cfbb72d7f'] = 'Tienes derecho a recuperar los datos que has facilitado al sitio en cualquier momento. Haz clic en "Obtener mis datos" para descargar automáticamente una copia de tus datos personales en un archivo PDF o CSV.';
+$_MODULE['<{psgdpr}prestashop>customerpersonaldata17_915f4446a47b51378e37374cfbb72d7f'] = 'En cualquier momento, tiene derecho a recuperar los datos que ha proporcionado a nuestro sitio. Haga clic en "Obtener mis datos" para descargar automáticamente una copia de sus datos personales en un archivo PDF o CSV.';
 $_MODULE['<{psgdpr}prestashop>customerpersonaldata17_49b5d04f5f20820830f080d4674c8669'] = 'OBTENER MIS DATOS CSV';
 $_MODULE['<{psgdpr}prestashop>customerpersonaldata17_7b0b06bfe69473081553bcccb12ca068'] = 'OBTENER MIS DATOS PDF';
-$_MODULE['<{psgdpr}prestashop>customerpersonaldata17_91a49c1d45a5d37e85658d01b6f37423'] = 'Solicitudes de rectificación y supresión';
-$_MODULE['<{psgdpr}prestashop>customerpersonaldata17_01cb324ae91e9bd16213cc2945334249'] = 'Tienes derecho a modificar todos los datos personales que aparecen en la página "Mi cuenta". Para cualquier otra solicitud que puedas tener respecto a la rectificación o supresión de tus datos personales, ponte en contacto con nosotros a través de nuestra';
+$_MODULE['<{psgdpr}prestashop>customerpersonaldata17_91a49c1d45a5d37e85658d01b6f37423'] = 'Solicitud de modificación y borrado';
```

Comment:
```suggestion
$_MODULE['<{psgdpr}prestashop>customerpersonaldata17_91a49c1d45a5d37e85658d01b6f37423'] = 'Solicitud de rectificación y supresión';
```
Esos son los términos correctos según https://europa.eu/youreurope/business/dealing-with-customers/data-protection/data-protection-gdpr/index_es.htm